cmd/contour/shutdownmanager.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 const (
-	prometheusURL      = "http://unix/stats/prometheus"
+	prometheusURL      = "http://unix/stats/prometheus?filter=http\\..*\\.downstream_cx_active"
```

Comment:
nit: might be good to note with a comment that this filters on the "normal" stats name, not the prometheus sanitized one, the regex matches against the below

```
http.admin.downstream_cx_active: 10
http.envoy-admin.downstream_cx_active: 1
http.ingress_http.downstream_cx_active: 3
http.stats.downstream_cx_active: 0
```

and not the prometheus format:

```
envoy_http_downstream_cx_active{envoy_http_conn_manager_prefix="admin"} 10
envoy_http_downstream_cx_active{envoy_http_conn_manager_prefix="envoy-admin"} 1
envoy_http_downstream_cx_active{envoy_http_conn_manager_prefix="ingress_http"} 4
envoy_http_downstream_cx_active{envoy_http_conn_manager_prefix="stats"} 0
```

Reply:
Also since by default the regex match is a substring match, would be nice to further optimize and include the line start and end anchors to make the regex:

```suggestion
	prometheusURL      = "http://unix/stats/prometheus?filter=^http\\..*\\.downstream_cx_active$"
```

Reply:
Thanks. I've applied both of your suggestions.